homeassistant/components/binary_sensor/rest.py

```diff
@@ -23,7 +23,7 @@
 MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=60)
 
 
-# pylint: disable=unused-variable
+# pylint: disable=unused-variable, logging-too-many-args
```

Comment:
Don't disable this check. It is only fired when you are actually giving too many args to logging based on the number of  `%s` in your logging message.